oneview_redfish_toolkit/conf/redfish.conf

```diff
@@ -22,7 +22,7 @@ DeliveryRetryAttempts = 3
 DeliveryRetryIntervalSeconds = 30
 
 [ssl]
-SSLType = self-signed
+SSLType =
```

Comment:
Why this configuration change.?

Reply:
I'm not sure why this is picked as a change. This feature is retained as is.